src/app/shared/dummy.service.ts

```diff
@@ -106,21 +106,19 @@ export class DummyService {
   readonly PROCESS_TEMPLATES: ProcessTemplate[] = [
     {
       name: 'Agile',
-      description: `Agile is self-organizing cross-functional teams, that focus on planning, evolutionary development, 
-      early delivery, and continuous improvement.`
+      description: `Agile-based planning (more generic than Scrum)`
     },
     {
       name: 'Scrum',
-      description: `Scrum is an iterative and incremental agile software development framework for managing product development.`
+      description: `Scrum-based planning.`
```

Comment:
@pmuir Please update this line to the new line in the doc that @catrobson updated.